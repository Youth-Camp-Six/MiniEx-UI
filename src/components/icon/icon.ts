import * as MiIcon from 'react-feather';
export { MiIcon }