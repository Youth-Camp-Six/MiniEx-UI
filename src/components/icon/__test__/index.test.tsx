/**
 * @jest-environment jsdom
 */

import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { MiIcon } from '../icon';

describe('MiIcon', () => {
  // 快照测试
  it('test MiIcon', () => {
    const wrapper = render(
      <div>
        <MiIcon.Activity></MiIcon.Activity>
        <MiIcon.Anchor></MiIcon.Anchor>
      </div>
    );
    expect(wrapper.container.firstChild).toMatchSnapshot();
  });
});
