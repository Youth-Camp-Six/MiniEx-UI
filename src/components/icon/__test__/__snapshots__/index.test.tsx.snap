// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MiIcon test MiIcon 1`] = `
<div>
  <svg
    fill="none"
    height="24"
    stroke="currentColor"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
    viewBox="0 0 24 24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <polyline
      points="22 12 18 12 15 21 9 3 6 12 2 12"
    />
  </svg>
  <svg
    fill="none"
    height="24"
    stroke="currentColor"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
    viewBox="0 0 24 24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      cx="12"
      cy="5"
      r="3"
    />
    <line
      x1="12"
      x2="12"
      y1="22"
      y2="8"
    />
    <path
      d="M5 12H2a10 10 0 0 0 20 0h-3"
    />
  </svg>
</div>
`;
